bundles/net.tourbook/src/net/tourbook/map3/view/Map3View.java

```diff
@@ -2522,12 +2476,12 @@ private void updateModifiedTours(final ArrayList<TourData> modifiedTours) {
 
    private void updateTrackSlider() {
 
-      final TrackSliderLayer trackSliderLayer = getLayerTrackSlider();
-      if (trackSliderLayer == null) {
-         return;
-      }
+//      final TrackSliderLayer trackSliderLayer = getLayerTrackSlider();
+//      if (trackSliderLayer == null) {
+//         return;
+//      }
 
```

Comment:
Because if you look the variable `trackSliderLayer` is not used below as the code is [also commented](https://github.com/wolfgang-ch/mytourbook/blob/4d46f667da3a7c4e44549d33a5fdd9b7575029c6/bundles/net.tourbook/src/net/tourbook/map3/view/Map3View.java#L2532)
// ???		trackSliderLayer.setSliderVisible(false);

Reply:
I have reverted to your code and only added my code.

Reply:
> Because if you look the variable `trackSliderLayer` is not used below as the code is [also commented](https://github.com/wolfgang-ch/mytourbook/blob/4d46f667da3a7c4e44549d33a5fdd9b7575029c6/bundles/net.tourbook/src/net/tourbook/map3/view/Map3View.java#L2532)

This variable is not used but `getLayerTrackSlider()` returns `null` also, when the layer is disabled (see javadoc) and then `updateTrackSlider()` should also do nothing

```

   /**
    * @return Returns {@link TrackSliderLayer} or <code>null</code> when layer is not displayed.
    */
   private TrackSliderLayer getLayerTrackSlider() {

      final TrackSliderLayer layer = Map3Manager.getLayer_TrackSlider();

      if (layer.isEnabled() == false) {
         // layer is not displayed
         return null;
      }

      return layer;
   }

```